HelloWorld/HelloWorld/Program.cs

```diff
@@ -3,15 +3,16 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Helpers;
```

Comment:
That looks good, but now that I see the new Helper library added, I notice you've got some unnecessary using statements here. Our coding standards require that all unnecessary using statements be removed.